sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor.cs

```diff
@@ -1088,6 +1094,7 @@ protected internal virtual async Task ValidateProcessingPreconditions(Cancellati
         ///   be called instead.
         /// </remarks>
         ///
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
How did we get away with a "not implemented exception" here?   I swear that would have been a breaking change.